docs/source/openshift/exposing-services.adoc

```diff
@@ -59,8 +59,18 @@ $ mysql --user=root --password=admin --host=$(minishift ip) --port=30907
 [[port-forwarding]]
 == Port Forwarding
 
+=== Using `oc port-forward`
 If you want to quickly access a port of a specific pod of your cluster, you can also use the `oc port-forward` command of the xref:../openshift/openshift-client-binary.adoc#[OpenShift CLI].
 
 ----
 $ oc port-forward POD [LOCAL_PORT:]REMOTE_PORT
 ----
+
+=== Using VirtualBox tools
+In case you're using the `virtualbox` driver there is another method you can use for port forwarding. This method will allow for permanent port forwarding as well as for forwarding multiple ports at the same time. This method requires you to set up a `nodePort` as outlined above.
```

Comment:
Please put each sentence on its own line. This makes reviewing diffs significantly easier whenever changes are made. :)

Reply:
There's also no need to use monospace for VirtualBox, though we do have to write it with the appropriate capitalization. We're also attempting to be more accurate when referring to driver plug-ins by specifically naming them as such (instead of simply using "driver").